docs/development.md

```diff
@@ -0,0 +1,103 @@
+## Development
+> Guide to local development of this plugin
+
+### Installation
+
+#### Requirements
+
+- Ruby
+- Bundler
+
+#### Install system dependencies
+
+- Install Ruby - see the [Downloads](https://www.ruby-lang.org/en/downloads/) page.
+- Install Bundler - see [Bundler](https://bundler.io/) homepage.
+
+#### Clone
+
+Clone the repo, or your fork.
+
+```bash
+$ git clone git@github.com:jekyll/github-metadata.git
+$ cd github-metadata
+```
+
+#### Install project dependencies
+
+Configure Bundler.
+
+```bash
+$ bundle config set --local path vendor/bundle
+```
+
+Install gems.
+
+```bash
+$ bundle install
+```
+
+Or, for a faster install.
+
+```bash
+$ script/bootstrap
+```
+
+## Usage
+
+See the [script](/script/) directory.
+
+### Format
+
+Check formatting.
+
+```bash
+$ script/fmt
+```
+
+Fix formatting issues.
+
+```bash
+$ script/fmt -a
+```
+
+### Open interactive console
+
+```bash
+$ script/console
+```
+
+### Test
+
+Run all unit tests:
+
+```bash
+$ script/test
+```
+
+Run specific unit tests.
+
+```bash
+$ script/test PATH
+$ # e.g.
+$ script/test spec/owner_spec.rb
+```
+
```

Comment:
May I suggest the following patterns as well? I found these useful when I started contributing to this project and had to diagnose CI/CD failures and add new test cases. But for someone like myself who does not use Ruby or Rspec professionally, they are not obvious.

Run tests in the order they are written (not random).

```bash
$ script/test --order defined
```

Run tests in the same random order as a previous run.
```bash
$ script/test --seed 12345
```

Run tests with verbose trace logs.
```bash
$ script/test --format documentation
```

Reply:
Thanks, I've added.